src/Classes/ListingHydrator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace ListingsApp\Classes;
+
+/**
+ * Class ListingHydrator
+ * @package ListingsApp\Classes
+ */
+
+class ListingHydrator
+{
+    /**
+     * @param PDO $db
+     * @param string $type
+     * @return array
+     */
+    public static function getListingsByType(PDO $db, string $type): array
+    {
+        $query = $db->prepare('SELECT * FROM `listings` WHERE `type` = :type;');
```

Comment:
SELECT * is naughty! Although I believe Nate may have written a full version of the query along with aliases so might be worth seeing if you can grab it from him.

Reply:
Do we also not need to join with the statuses and types tables? 

Type is less important, but we will need the name of the status in order to display it on the listings pages.